### PR TITLE
Add builder for list of tuples

### PR DESCRIPTION
**Pull request by akegalj**

### CODE REVIEW

File: src/Serokell/Util/Text.hs [hunk 1]
```diff
@@ -105,6 +106,9 @@ tripleF = later tripleBuilder
 listJson :: (Foldable t, Buildable a) => Format r (t a -> r)
 listJson = later listBuilderJSON
 
+listMapTuple :: (Traversable t, Buildable a, Buildable b) => Format r (t (a, b) -> r)
+listMapTuple = later mapBuilder
```
Comment (Martoon-00) [inline src/Serokell/Util/Text.hs]: Why `listMapTuple`, if `t (a, b)` already corresponds to just "map"?

Comment (akegalj) [reply src/Serokell/Util/Text.hs]: Are you proposing to call it `listMap`?

Comment (Martoon-00) [reply src/Serokell/Util/Text.hs]: Something like this seems more reasonable IMO

Comment (akegalj) [reply src/Serokell/Util/Text.hs]: Fixed

**Pull request merged**